_posts/en/newsletters/2018-12-18-newsletter.md

```diff
@@ -0,0 +1,200 @@
+    no user effort, they don't require withdrawing 100% of deposits from
+    an exchange to prove it can sign for 100% of reserves, and they
+    don't bloat the block chain or increase transaction fees.  Optech
+    suggests future initiatives focus on users requesting (and exchanges
+    providing) proof of reserves.
+
+- **Description about what might be included in a Schnorr/Taproot soft
+  fork:** Bitcoin protocol developer Anthony Towns has [posted][towns
+  schnorr taproot] a quite readable email describing what he thinks
+  ought to be included in a soft fork that adds the Schnorr signature
+  scheme plus Taproot-style MAST to Bitcoin.  These are just his
+  opinions and not a definitive proposal, but they are in line with what
+  Optech has heard other developers discussing.  As a reminder, even a
+  definite proposal with code can only truly become part of the Bitcoin
+  protocol if it has the support of a large number of economically
+  relevant full node users---and building such support could take years,
+  if it's achieved at all.
+
+- **LN protocol IRC meeting:** protocol developers for LN have agreed to
+  try converting their periodic meeting for developing the LN
+  specification from a Google Hangout to an IRC meeting after receiving
+  requests from several developers.  The [next meeting][ln irc meeting]
+  will be Tuesday, January 8th, at 19:00 (UTC).
+
+## Notable code changes
+
+*Notable code changes this week in [Bitcoin Core][core commits],
+[LND][lnd commits], [C-lightning][cl commits], and [libsecp256k1][secp
+commits].*
+
+- [Bitcoin Core #14573][] moves various miscellaneous options that
+  opened separate dialogues in the Bitcoin-Qt GUI to a new top-level
+  menu item labeled *Window*, hopefully making those options easier to
+  find and use.
+
+- [LND #1984][] adds a new `listunspent` RPC that lists each of the
+  wallet's unspent outputs.  It can take two parameters: (1) the minimum
+  number of confirmations the unspent output must have or (2) the maximum
+  it can have.  The minimum can be set to `0` to print unconfirmed
+  outputs.
+
+- [LND #2039][] adds the ability to get the status of the autopilot
+  functionality as well as allowing enabling or disabling it while the
+  program is running.  Autopilot is the ability of the software to
+  automatically suggest new channels to open when a user is first
+  connecting to LN or wants additional spending capacity.
+
+- [LND #2324][] updates and expands LND's installation documentation.
+
+- [C-Lightning #2155][] disables by default the
+  `option_data_loss_protect` feature described in the notable commits of
+  [newsletter #10][].  The feature wasn't working reliably, so it will
+  only be enabled for users that opt-in to experimental features.
+
+- [C-Lightning #2154][] now allows plugins to send log notifications
+  that will be written to the lightningd's log files.
+
+- [C-Lightning #2132][] adds significant inline code documentation for
```

Comment:
I think documentation and commenting are very important, but I'm not convinced that this makes the bar for notable code changes.